python/cuml/test/test_pickle.py

```diff
@@ -160,7 +164,12 @@ def assert_model(pickled_model, X_test):
         # Confirm no crash from score
         pickled_model.score(X_test, np.zeros(X_test.shape[0]))
 
-    pickle_save_load(tmpdir, create_mod, assert_model)
+    try:
+        pickle_save_load(tmpdir, create_mod, assert_model)
+
+    except NotImplementedError:
+        pytest.xfail("Pickling for multi-class classification is not "
```

Comment:
Instead of xfail, you can use pytest to say "we better get an exception here" like:
```
 with pytest.raises(NotImplementedError):
    ... do something ...
```